txpipe/twopoint_fourier.py

```diff
@@ -221,11 +223,85 @@ def load_maps(self):
         syst_nc = None
         syst_wl = None
 
+        
+        # Load HEALPix systematics maps
+        syst_nc = self.config['syst_nc']
+        syst_wl = self.config['syst_wl']
+        if syst_nc:  
+            print('Deprojecting systematics maps for number counts')
+            n_systmaps = 0
+            s_maps = []
+            systmaps_dir = self.config['systmaps_nc_dir']
+            systmaps_path = pathlib.Path(systmaps_dir)
+            try:
+                for systmap in systmaps_path.iterdir():
+                    if systmap.is_file():
+                        systmap_file = str(systmap)
+                        print('Reading nc systematics map file: ', systmap_file)
+                        syst_map = healpy.read_map(systmap_file,verbose=False)
+                        
+                        # Find value at given ra,dec
+                        ra = 55.
+                        dec = -30.
+                        theta = 0.5 * np.pi - np.deg2rad(dec)
+                        phi = np.deg2rad(ra)
+                        nside = healpy.pixelfunc.get_nside(syst_map)
+                        ipix = healpy.ang2pix(nside, theta, phi)
+                        print('Syst map: value at ra,dec = 55,-30: ', syst_map[ipix])
+
+                        # normalize map for Namaster
+                        # set pixel values to value/mean - 1
+                        syst_map_mask = syst_map != healpy.UNSEEN
```

Comment:
are there typically any nans in these maps to worry about?

Reply:
There should not be any NaN's in the HELAPix map as far as I know.